proto/sentry_protos/snuba/v1/endpoint_trace_item_attributes.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+//A request for "which queryable attributes are available for these projects between these dates"
+// - used for things like autocompletion
+// EAP does not make a distinction between data stored as columns on a table vs attributes stored
+// dynamically (e.g. tags, contexts, etc). To the user of the interface, they are the same
+message TraceItemAttributesRequest {
```

Comment:
The naming of this and `proto/sentry_protos/snuba/v1/endpoint_trace_item_columns.proto`
confuse me a bit, so might be misunderstanding

Here's my current guess;

TraceItemAttributesRequest -> Gets me the names of attrs, excluding columns?
TraceItemColumnsRequest -> Gets the columns, excluding attrs?
AttributeValuesRequest -> Gets me the values for a given attr or column

Reply:
I can see how this is confusing. I renamed 

`TraceItemAttributesRequest -> TraceItemAttributeNamesRequest`
`AttributeValuesRequest -> TraceItemAttributeValuesRequest`

For  `TraceItemAttributesRequest` I could use some help. This is the endpoint that is intended to serve tabular queries. Like your more standard `SELECT project_id, duration, span_name FROM spans` hence I named it columns. I'm open to other ideas of how to name this